src/params.h

```diff
@@ -201,20 +201,20 @@
  * its own purposes.  This does not affect password cracking speed after the
  * loading is complete.
  */
-#define PASSWORD_HASH_SIZE_FOR_LDR	5
+#define PASSWORD_HASH_SIZE_FOR_LDR	4
 
 /*
  * Hash table sizes.  These may also be hardcoded into the hash functions.
  */
 #define SALT_HASH_LOG			20
 #define SALT_HASH_SIZE			(1 << SALT_HASH_LOG)
-#define PASSWORD_HASH_SIZE_0		0x10
-#define PASSWORD_HASH_SIZE_1		0x100
-#define PASSWORD_HASH_SIZE_2		0x1000
-#define PASSWORD_HASH_SIZE_3		0x10000
-#define PASSWORD_HASH_SIZE_4		0x100000
-#define PASSWORD_HASH_SIZE_5		0x1000000
-#define PASSWORD_HASH_SIZE_6		0x8000000
+#define PASSWORD_HASH_SIZE_0		0x100
+#define PASSWORD_HASH_SIZE_1		0x1000
+#define PASSWORD_HASH_SIZE_2		0x10000
+#define PASSWORD_HASH_SIZE_3		0x100000
+#define PASSWORD_HASH_SIZE_4		0x1000000
+#define PASSWORD_HASH_SIZE_5		0x8000000
```

Comment:
Here is the bug.  v5 is 8x bigger.

Reply:
How is that a problem? It's intentional.

Reply:
No bug there. The 8x is intentional, and besides it's not new with this merge. The thresholds are also in their proper order - although the divisors vary, they don't vary so much as to make a higher-numbered threshold lower than a lower-numbered one. And even if this were the case, nothing would fail - things would just be suboptimal.